### DISCUSSION TITLE
Do I have a NodeMCU12E?

### DISCUSSION THREAD

**Discussion by WesWesthaver** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#608] [Q&A]
The Lay-Z-Spa Module app says:

 Firmware
Only update if:
You have a NodeMCU12E

Being new to this, I'd like to know how to determine if I have the required "NodeMCU12E" to allow for over-the-air firmware updates.

[Picture of my Lay-Z-Spa Module](http://www.westhaver.com/SaluSpa/IMG_20230822_193805.jpg)

-Wes

**Comment by visualapproach** [comment]
Can't see picture

**Reply by visualapproach** [reply]
Yes you're good to go
